### PR TITLE
feat(deployment): support list of imagePullSecrets #major

### PR DESCRIPTION
**Pull request by xeivieni**
This chart supports only one image pull secret instead of a list for the deployment.yml manifest. This Pull Request aims at supporting a list of secrets.

### CODE REVIEW

**Comment by aslafy-z** [comment]
I just merged a bunch of updates to the readme and the values file (https://github.com/stakater/application/pull/335). Can you please update your PR with these changes?

File: application/Chart.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ type: application
 
 # Helm chart Version
 
-version: 4.2.12
+version: 4.2.13
```
Comment (aslafy-z) [inline application/Chart.yaml:9]: Can you please revert this change? The chart version will be automatically bumped.

**Review by aslafy-z — CHANGES_REQUESTED** [review]

File: application/Chart.yaml (hunk 1 above)
Comment (xeivieni) [reply application/Chart.yaml]: done ;)

**Review by aslafy-z — APPROVED** [review]

**Comment by aslafy-z** [comment]
This was released with https://github.com/stakater/application/releases/tag/v5.0.0. Thank you for your contribution.
